app/src/main/report/components/Sidebar/UserList.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react"
```

Comment:
Right now this component is very simple, but in a future task we will add functionality to add/remove users from the list